.github/workflows/lint.patch.yml

```diff
@@ -1,10 +1,9 @@
 name: Lint Rust files
 
 on:
-  push:
+  pull_request:
     branches:
-      - "**"
-      - "!main"
+      - 'main'
     paths-ignore:
```

Comment:
Only PR? Just confirming

Reply:
Yes, there was no value added on using it on push, and the `reviewdog/action-actionlint@v1.21.0` required the triggering event to be a PR, and making another workflow just for this was too cumbersome (until we find a better approach).

Reply:
Yes